jobmaster/dm/metadata/job.go

```diff
@@ -19,7 +19,9 @@ const (
 	StageRunning
 	StagePaused
 	StageFinished
-	StageDelete
+	// UnScheduled means the task is not scheduled.
+	// This usually happens when the worker is offline.
+	StageUnscheduled
```

Comment:
I'm not sure what's the difference of stage and status and other term. @liuzix could your give some advice on naming?

Reply:
This stage only use in dm-master

Reply:
`stage` seems to imply irreversibility, while `status` implies a state machine.

Reply:
Will CDC also need this concept? we can find a better name at present.